homework-17/mocha-test/calculator.spec.ts

```diff
@@ -0,0 +1,82 @@
+import assert from "assert";
+import { expect } from "chai";
+import { Calculator } from "../src/calculator";
+
+let calculator: any;
+
+const VALIDATION_ERROR = "The result is not as expected!";
```

Comment:
You use this constant here and in the `jest` tests. Please create a separate file for constants and place this constant there.